include/albatross/src/models/sparse_gp.hpp

```diff
@@ -370,9 +370,10 @@ class SparseGaussianProcessRegression
   // };
 
   // TODO(@peddie) SPQR configuration
```

Comment:
what does this TODO mean?

Reply:
Hmm . . . that's a good question.  I think it's outdated, since `SPQR_create()` sets pivot tolerance and thread count.